middleware/validateProductData.js

```diff
@@ -30,11 +30,9 @@ function validateProductInput (data) {
   }
 
   // stock checks
-  if (!stock) {
-    errors.stock = 'Stock field is required'
-  } else if (isNaN(stock)) {
+  if (stock && isNaN(stock)) {
     errors.stock = 'Stock must be a number'
-  } else if (Number(stock) <= 0) {
+  } else if (stock && Number(stock) <= 0) {
     errors.stock = 'Stock cant be less than 0'
```

Comment:
@Rexben001 Can you help change this error message. It says stock can't be less than 0. Should say something like Stock can't be less than 1, or must be greater than 0. It was my mistake.